crates/payload/basic/src/lib.rs

```diff
@@ -787,6 +797,7 @@ pub struct BuildArguments<Attributes, Payload: BuiltPayload> {
     pub cancel: CancelOnDrop,
     /// The best payload achieved so far.
     pub best_payload: Option<Payload>,
+    pub is_resolving: Arc<AtomicBool>,
```

Comment:
this also needs a oneline doc what this represents

Reply:
done